### PR TITLE
flaskify

### PR DESCRIPTION
**Pull request by stanbar**
Add flask http server

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from flask_wtf import FlaskForm
+from wtforms import SubmitField
+from wtforms import IntegerField
+from wtforms.validators import DataRequired
+from flask_bootstrap import Bootstrap
+from coil import normalcoil
+
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'jakis ciag znakow'
```
Comment (stanbar) [inline app.py]: I don't recognize this language. Please use human language (english)

File: app.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        return redirect(url_for('index'))
+    return render_template(
+            'index.html',
+            form=form,
+            wrapsForm=session.get('wrapsForm'),
+            innerDiameterForm=session.get('innerDiameterForm'), 
+            wireDiameterForm=session.get('wireDiameterForm'), 
+            legsLengthForm=session.get('legsLengthForm'), 
+            known=session.get('known', False)
+            )
```
Comment (stanbar) [inline app.py]: This service is going to be API microservice, so these views are redundant. Please consider sending just HTTP POST with json data over curl.

File: app.py (hunk 1 above)
Comment (mozerpol) [reply app.py]: language of thieves and drunks

File: app.py (hunk 2 above)
Comment (mozerpol) [reply app.py]: It's only for tests, for my purposes

**Comment by stanbar** [comment]
I suggest you to add shell script to testing the endpoint. Something like
`sendGenerateMesh.sh`:
```
#!/bin/sh

curl http://localhost:5000/generate-mesh \
-X POST \
-H "Content-Type: application/json" \
-d '{"wraps": 5, "innerDiameter": 3.2 , "wireDiameter": 0.322, "legsLength": 15 }'
```

**Pull request merged**